gridcapa-core-cc-app/src/test/java/com/farao_community/farao/gridcapa_core_cc/app/postprocessing/FileExporterHelperTest.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+
+package com.farao_community.farao.gridcapa_core_cc.app.postprocessing;
+
+import com.farao_community.farao.data.crac_api.Crac;
+import com.farao_community.farao.data.crac_creation.creator.fb_constraint.crac_creator.FbConstraintCreationContext;
+import com.farao_community.farao.data.crac_io_api.CracImporters;
+import com.farao_community.farao.data.rao_result_api.RaoResult;
+import com.farao_community.farao.data.rao_result_json.RaoResultImporter;
+import com.farao_community.farao.gridcapa_core_cc.api.resource.CoreCCFileResource;
+import com.farao_community.farao.gridcapa_core_cc.api.resource.HourlyRaoRequest;
+import com.farao_community.farao.gridcapa_core_cc.api.resource.HourlyRaoResult;
+import com.farao_community.farao.gridcapa_core_cc.api.resource.InternalCoreCCRequest;
+import com.farao_community.farao.gridcapa_core_cc.app.services.FileImporter;
+import com.farao_community.farao.minio_adapter.starter.MinioAdapter;
+import com.powsybl.iidm.network.Network;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * @author Thomas Bouquet {@literal <thomas.bouquet at rte-france.com>}
+ */
+public class FileExporterHelperTest {
+
+    private String newtorkFile = "/util/TestCase12NodesHvdc.uct";
+    private Path networkPath = Paths.get(getClass().getResource(newtorkFile).getPath());
+    private Network network = Network.read(networkPath);
+    private FileImporter fileImporter;
+    private MinioAdapter minioAdapter;
+    private InternalCoreCCRequest coreCCRequest;
+    private HourlyRaoResult hourlyRaoResult;
+    private HourlyRaoRequest hourlyRaoRequest;
+    private Boolean networkIsUploaded;
+    private Boolean raoResultIsUploaded;
+    private Boolean metadataIsUploaded;
+    private Boolean cneIsUploaded;
+    private String instantString = "2023-07-25T16:57:00Z";
+    private Instant instant = Instant.parse(instantString);
+    private OffsetDateTime timestamp = OffsetDateTime.of(2023, 7, 27, 10, 47, 51, 0, ZoneId.of("Europe/Brussels").getRules().getOffset(LocalDateTime.now()));
```

Comment:
did you migrate rao i tests ?

Reply:
the new tests rely on new input files but cover the same features